x-pack/plugins/elastic_assistant/server/routes/prompts/find_route.ts

```diff
@@ -44,13 +44,14 @@ export const findPromptsRoute = (router: ElasticAssistantPluginRouter, logger: L
         try {
           const { query } = request;
           const ctx = await context.resolve(['core', 'elasticAssistant', 'licensing']);
-          const license = ctx.licensing.license;
```

Comment:
Another spot that only had license check and not user, just want to make sure this was not intentional?